### PR TITLE
update link to docs folder

### PR DESCRIPTION
**Pull request by danlourenco**
Hey @dayhaysoos, @angiejones ! This PR is a follow-up on [this issue](https://github.com/TBD54566975/developer.tbd.website/issues/301). So, after exploring the Docusaurus docs, I'm not sure using the `editUrl` [config field](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#editUrl) is going to work without some rethinking of the UI. According to the docs, configuring the field will automatically add the "Edit this page" link you see on most Docusaurus sites. See below: 
Screenshot 2023-07-28 at 11 02 27 PM

Screenshot 2023-07-28 at 10 57 37 PM

Given that the TBD site has a custom Feedback Widget that contains the "Edit this page" link, I don't think this would play nice with the current designs.

Might need to think on this further, but in the meantime, I tweaked the URL to at least take you to the docs folder in GitHub instead of hitting the 404. Let me know what you think.

### CODE REVIEW

**Review by angiejones — APPROVED** [review]

**Comment by angiejones** [comment]
> @danlourenco is attempting to deploy a commit to the **TBD54566975** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=TBD54566975&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%225285c459b63048d09e22d1144f36677a8f752761%22%7D%2C%22id%22%3A%22Qma4yV7HAzCX5eGeL6GYqz7L1NinYgdxPKih1axyo7utiV%22%2C%22org%22%3A%22TBD54566975%22%2C%22prId%22%3A624%2C%22repo%22%3A%22developer.tbd.website%22%7D).

@dayhaysoos any idea what's this about?

**Comment by dayhaysoos** [comment]
> > @danlourenco is attempting to deploy a commit to the **TBD54566975** Team on [Vercel](https://vercel.com).
> > A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=TBD54566975&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%225285c459b63048d09e22d1144f36677a8f752761%22%7D%2C%22id%22%3A%22Qma4yV7HAzCX5eGeL6GYqz7L1NinYgdxPKih1axyo7utiV%22%2C%22org%22%3A%22TBD54566975%22%2C%22prId%22%3A624%2C%22repo%22%3A%22developer.tbd.website%22%7D).
> 
> @dayhaysoos any idea what's this about?

TL;DR we are having some issues with Netlify so ALR temporarily set us up for Vercel as a "just in case".

I'll give you more details on slack
